server/src/main/java/io/seata/server/coordinator/DefaultCore.java

```diff
@@ -82,6 +82,7 @@ public Long branchRegister(BranchType branchType, String resourceId, String clie
                 globalSession.addBranch(branchSession);
             } catch (RuntimeException ex) {
                 branchSession.unlock();
+                LOGGER.error("Failed to add branchSession to globalSession : " + ex);
```

Comment:
Please use LOGGER.error("Failed to add branchSession to globalSession:{}",ex.getMessage(),ex);

Reply:
Ok,Thank for your patience,I have modified it,but,I read a lot of error logs in seata,The style is not same.for example:
LOGGER.error("transaction[" + tx.getXid() + "] retry fetch status times exceed the limit [" + RETRY_MAX_TIMES + " times]");
LOGGER.error("fetch GlobalTransaction status error", e);
LOGGER.error("error occurred while getting value{}", e.getMessage());
LOGGER.error("not support metrics exporter type: " + exporterTypeName, exx);
......
they all used at catch{LOGGER.erro},unify the writing style will be better.

Reply:
Yes.The problem does exist,will optmize in the future.